source/data/tier1data/features/automation.rst

```diff
@@ -0,0 +1,83 @@
+###################
+Workflow automation
+###################
+
+************************
+Why workflow automation?
+************************
+
+Research data management covers a lot of tasks, some of which are
+repetitive. For example, a workflow might include pre-processing of
+files, adding certain metadata, or emptying a certain directory
+regularly.
+
+Automating repetitive tasks that are usually done manually, has two
+goals. First of all, it can save users a lot of time. Secondly, it
+eliminates human error. When a user needs to apply the same action to a
+lot of files, mistakes are bound to happen.
+
+By using scripts, we can automate a lot of these processes, keeping our
+hands free for other tasks.
+
+*************************
+How to automate workflows
+*************************
+
+
+Client-side automation
+======================
+
+Users can automate workflows themselves on the client-side. 
+We have two clients which can be integrated easily into your HPC workflow:
+- :ref:`iCommands<icommands>` are bash commands and can thus be integrated into your job scripts
+- Users who use Python in their HPC jobs can integrate the :ref:`Python client<python-client>` in their script
+This way, you can easily retrieve data to be used in calculations, store results, add metadata and do other tasks automatically when you submit a job.
+
+Of course, these clients can also be used outside of the HPC environment.
+
+
+
+Machine accounts
+----------------
+
+By default, authentication to Tier-1 Data is limited to 60 hours (via the Python client) or 168 hours (via iCommands).
+However, some automated processes require longer authentication.
+
+For these cases, we created machine accounts: these are accounts that authenticate with a token, which lasts for 4 years.
+To activate these machine accounts, go to the page of your project on the `ManGO portal <https://mango.vscentrum.be/>`_, 
```

Comment:
Mention that this task can only be done by the managers

Reply:
@ingridbr Group management pages aren't added now, but in a next iteration. So I don't have any manager definition to refer to.

Reply:
Ok!